packages/cmrjs/src/cmr-utils.js

```diff
@@ -1090,8 +1090,13 @@ async function getUserAccessibleBuckets(edlUser, cmrProvider = process.env.cmr_p
  *    available.
  */
 async function getGranuleTemporalInfo(granule) {
-  const cmrFile = granuleToCmrFileObject(granule);
-  if (cmrFile.length === 0) return {};
+  let cmrFile = [];
+  try {
+    cmrFile = granuleToCmrFileObject(granule);
+  } catch (error) {
+    log.debug(`getGranuleTemporalInfo failed to granuleToCmrFileObject ${JSON.stringify(granule)}, ${error.message}`);
```

Comment:
So the intended change here to not throw in cases where an invalid file object comes in?    

Upstream a bunch of things use `granulesToCmrFileObjects` - is the error case not valid for any of them?

Reply:
granuleToCmrFileObject is not modified.